src/persist-client/src/cache.rs

```diff
@@ -113,6 +129,11 @@ impl PersistClientCache {
         &self.cfg
     }
 
+    /// Returns persist's [ConfigSet].
+    pub fn dyncfgs(&self) -> &ConfigSet {
```

Comment:
We shouldn't need this. `PersistConfig` has a `Deref` impl to `ConfigSet`

Reply:
Removed